diff --git a/node_modules/getenv/index.js b/node_modules/getenv/index.js
index 5e83c8f..c9e4d50 100644
--- a/node_modules/getenv/index.js
+++ b/node_modules/getenv/index.js
@@ -54,12 +54,12 @@ const convert = {
     return +value;
   },
   bool: function(value) {
-    const isBool = value === 'true' || value === 'false';
+    const isBool = (value || '').toLowerCase() === 'true' || (value || '').toLowerCase() === 'false';
     if (!isBool) {
       throw new Error('GetEnv.NoBoolean: ' + value + ' is not a boolean.');
     }
 
-    return value === 'true';
+    return (value || '').toLowerCase() === 'true';
   },
   boolish: function(value) {
     try {
